paddle/fluid/inference/api/paddle_inference_api.h

```diff
@@ -120,26 +117,49 @@ struct NativeConfig : public PaddlePredictor::Config {
   // GPU related fields.
   bool use_gpu{false};
   int device{0};
-  float fraction_of_gpu_memory{-1.f};  // Negative to notify initialization.
-  // NOTE: NOT use it, just for the internal test, will discard later
-  bool _use_mkldnn{false};
-  // Specify the variable's name of each input.
-  bool specify_input_name{false};
+  float fraction_of_gpu_memory{-1.f};  // Change to a float in (0,1] if needed.
 
+  // Specify the exact path of program and parameter files.
   std::string prog_file;
   std::string param_file;
+
+  bool specify_input_name{false};  // Specify the variable's name of each input.
```

Comment:
explain what will happen if false? This seems duplicate with the PaddleTensor.name